laterpay/asset_sources/js/laterpay-account-links.js

```diff
@@ -0,0 +1,36 @@
+// render LaterPay login / logout links using the LaterPay YUI dialog manager library
+YUI().use('node', 'laterpay-dialog', 'laterpay-iframe', 'laterpay-easyxdm', function(Y) {
+
+    // check, if all required variables are available
+    if (!lpAccountLinksUrl || !lpAccountNextUrl) {
+        return;
+    }
+
+    // define URLs to forward the user to after login / logout / registration
+    var loginLink  = 'https://web.laterpay.net/auth/user/login?_on_complete=' + lpAccountNextUrl,
```

Comment:
Additionally: [render_account_links()](https://github.com/laterpay/laterpay-wordpress-plugin/pull/548/files#diff-873977fbfe0208553a226a29e06dd91dR50) seems to write the URLs to JavaScript as they are returned by the LaterPay PHP client. That probably means that they are not urlencoded properly so that they can be concatenated to the URL here by just using `+`.

That bug is however probably meaningless, because it will be fixed when you use the correct endpoints as listed above as the URLs need to be created, signed and encoded server-side.